blog/16-03-23-create-deploy-and-mint-your-eRC223-contract-on-celo-with-hardhat/create-deploy-and-mint-your-eRC223-contract-on-celo-with-hardhat.md

```diff
@@ -0,0 +1,472 @@
+
+
+
+## Deploying The ERC223 token standard
+
+In the next few step, you will deploy your ERC223 contract;
+
+1. First, create a new file and name the file .env and create a  variable name `MNEMONIC`
+2. Now, login to your preferred wallet extension on your browser to get your wallet MNEMONIC, and copy the entire phrase.
+***Note: Your wallet MNEMONICS is the same as the key phrase given to you when you initially created your wallet account. It usually ranges from 12 to 18 random words for verifying your ownership, like a passphrase***.
+
+3. Paste the MNEMONIC phrase as the value for the variable MNEMONIC inside your `.env` file.
+
+***Note: Your MNEMONICS should be kept secret and secure (Not public) at all points. DO NOT REVEAL this phrase at all times. No quotes should be added***.
+
+4. Now, go to your deploy.js script inside the `scripts` folder and copy and paste the code below into the file:
+
+```solidity
+const hre = require("hardhat");
+
+
+const main = async () => {
+  const MyToken = await hre.ethers.getContractFactory("MyToken");
+  const Token = await MyToken.deploy();
+  await Token.deployed();
+
+
+  console.log("The MyToken contract was deployed to: ", Token.address);
+};
+
+
+const runMain = async () => {
+  try {
+    await main();
+    process.exit(0);
+  } catch (error) {
+    console.error(error);
+    process.exit(1);
+  }
+};
+
+
+runMain();
+
+```
+
+5. Next, run the command `npx hardhat compile`, and your contract should be compiled without any error, with the result like the image below:
+
+![compiled](https://user-images.githubusercontent.com/69092079/225623148-edd9e69f-0ffb-4b08-874c-738b0f543ce3.jpg)
+
+
+6. Finally, run the command npx hardhat `$ npx hardhat run scripts/deploy.js --network alfajores`, and successful execution of the command should look like the image below:
+
+![deployed](https://user-images.githubusercontent.com/69092079/225623246-208979a5-9fde-44c5-86b3-769af4cc3611.png)
+
+
+## Conclusion​
+Overall, creating and deploying an ERC223 smart contract on Celo is an exciting opportunity for developers to explore the potential of decentralized finance (DeFi) and create innovative new applications that can revolutionize the financial landscape. As the blockchain industry continues to grow and evolve, the use of smart contracts on platforms like Celo will become increasingly important, and developers who understand how to leverage this technology will be well-positioned for success in the future.
+
+
+## Next Steps​
+Here are some other NFT-related tutorial articles you might be interested in reading about:
+
+* [Build an NFT collection on Celo](https://www.celosage.com/how-to-quickly-build-an-nft-collection-on-celo/)
+* [Deploy an ERC 721 contract using Tatum API](https://www.celosage.com/how-to-deploy-an-erc721-smart-contract-using-the-tatum-api/)
```

Comment:
We are supposed to leave one line of space after each of the heading or subheading